photography/__manifest__.py

```diff
@@ -17,6 +17,7 @@
         'knowledge',
         'project_enterprise',
         'website_appointment',
+        'web_studio',
```

Comment:
Hmm, not sure we want that for a web page. We might need to change the condition. Could you update the test to handle this case? (only a ir.ui.view for the website should not require studio)